server/battleships/use_cases/implementation/src/main/java/eu/optas/use_cases/implementation/ShootInteractor.java

```diff
@@ -9,6 +9,7 @@
 import eu.optas.use_cases.api.ShootUC;
 import eu.optas.utils.GameNotFoundException;
 import eu.optas.utils.GameState;
+import kotlin.Pair;
```

Comment:
why kotlin?

Reply:
Jeigu čia tik dėl to, kad vienoje vietoje nori laikyti tiek tą updatinamą žaidimą/ship tai paprasčiausiai pasidaryk inner klasę, kurioje laikytum tuos objektus.